widget/popup.go

```diff
@@ -90,10 +90,9 @@ func (p *PopUp) CreateRenderer() fyne.WidgetRenderer {
 		return &modalPopUpRenderer{popUp: p, bg: bg, objects: []fyne.CanvasObject{bg, p.Content}}
 	}
 
-	shadow := newShadow(shadowAround, theme.Padding()*2)
 	bg := canvas.NewRectangle(theme.BackgroundColor())
-	objects := []fyne.CanvasObject{shadow, bg, p.Content}
-	return &popUpRenderer{popUp: p, shadow: shadow, bg: bg, objects: objects}
+	objects := []fyne.CanvasObject{bg, p.Content}
+	return &popUpRenderer{shadowingRenderer: newShadowingRenderer(objects, 4), popUp: p, bg: bg}
```

Comment:
I think this was in the discussion, but use a const for the magic '4'

Reply:
ack